tools/interactive/interactive_pampa.xml

```diff
@@ -0,0 +1,55 @@
+<tool id="interactive_tool_pampa" tool_type="interactive" name="pampa" version="0.0.1" profile="22.01">
+    <description>Performance Indicators of Marine Protected Areas for the management of coastal ecosystems, resources and their uses</description>
+    <requirements>
+        <container type="docker">ylebras/pampa:latest</container>
```

Comment:
Always remember to version those images when you bring them in production, ideally in sync with the tool version.